### PR TITLE
Update careers for 2021

### PR DESCRIPTION
**Pull request by Zejnilovic**

### CODE REVIEW

File: _careers/big_data_eng.md [hunk 1]
```diff
@@ -2,40 +2,45 @@
...
+-	Design and implementation of generic tools and platforms for some of the following:
+  - enabling simple data ingestion and consumption of corporate data lake
+  - various streaming use cases and products
+  - machine learning applications and pipelines
+-	Contribute to our projects, which can be found on [GitHub](https://github.com/AbsaOSS)
+  - we also contribute to others, like Apache Spark
+-	Research of new technologies and investigation of their usability in our Big Data platforms
+-	Filling gaps in the big data eco-system for it to be fully usable by financial and other highly regulated industries
+-	Presenting R&D work at commercial and academic conferences
+-	Mentoring junior big data engineers
```
Comment (kevinwallimann) [inline _careers/big_data_eng.md:22]: Are these intentional changes? The second-level bullet points are not indented anymore

File: _careers/ui_eng.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+- Plan and implement new ideas by helping with UX and UI designs for Big Data Applications, tools and platforms.
+- Develop reusable components and core frontend libraries.
+- Optimize existing user interface designs and contribute to our projects, which can be found on [GitHub](https://github.com/AbsaOSS).
+- Research of new technologies and investigation of their usability across the platform.
+- Come up with best practices for UI development and share knowledge and expertise with other team members.
+- Presenting R&D work at commercial and academic conferences. 
+
+### Must Have
+- Excellent skills with HTML5& CSS3.
+- At least 2 years of experience in UI web development with any modern JSframework (Angular4+, ReactJS, Vue.js, etc)
```
Comment (kevinwallimann) [inline _careers/ui_eng.md:22]: Typos: `JSframework` -> `JS framework`, `Angular4+` -> `Angular 4+`, `etc` -> `etc.`

File: _careers/ui_eng.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+- Understanding of the general concept of SPA state management.
+- Understanding of the HTTP protocol, what is REST API, and how the client-side communicates with a server-side.
+- Strong programming and problem-solving skills.
+- Ability to work independently as well as collaborate closely with developers, analysts etc. in an agile setting
+- Be eager to learn new approaches and technologies
+
+### Desirable Skills
+- Experiencewith:
+    - state management tools: NgRx, React Redux, etc.
+    - UX/UItools: Figma, AdobeXD, Sketch, etc.
```
Comment (kevinwallimann) [inline _careers/ui_eng.md:33]: Whitespace: `UItools` -> `UI tools`

File: _careers/ui_eng.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+- At least 2 years of experience in UI web development with any modern JSframework (Angular4+, ReactJS, Vue.js, etc)
+- At least 1 year of experience with Angular 4+, RxJS, Typescript, Jest/Jasmine.
+- Understanding of the general concept of SPA state management.
+- Understanding of the HTTP protocol, what is REST API, and how the client-side communicates with a server-side.
+- Strong programming and problem-solving skills.
+- Ability to work independently as well as collaborate closely with developers, analysts etc. in an agile setting
+- Be eager to learn new approaches and technologies
+
+### Desirable Skills
+- Experiencewith:
```
Comment (kevinwallimann) [inline _careers/ui_eng.md:31]: Whitespace: `Experiencewith` -> `Experience with`

File: _careers/ui_eng.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+    - state management tools: NgRx, React Redux, etc.
+    - UX/UItools: Figma, AdobeXD, Sketch, etc.
+    - CSS frameworks:Bootstrap, Materialize, Fondation, etc.
```
Comment (kevinwallimann) [inline _careers/ui_eng.md:34]: Whitespace: `rks:Boot` -> `rks: Boot`

File: _careers/ui_eng.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+    - Angular UI libraries:MaterialUI, PrimeNG, etc.
```
Comment (kevinwallimann) [inline _careers/ui_eng.md:35]: Whitespace: `libraries:MaterialUI` -> `libraries: Material UI`

File: _careers/ui_eng.md [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+    - CSS scripting language:SCSS, LESS, etc.
```
Comment (kevinwallimann) [inline _careers/ui_eng.md:36]: Whitespace: `language:SCSS` -> `language: SCSS`

File: _careers/big_data_eng.md (hunk 1 above)
Comment (Zejnilovic) [reply _careers/big_data_eng.md]: All fixed. Thank you. I did something with the whitespaces, I am not sure what.

File: _careers/ui_eng.md [hunk 8]
```diff
@@ -19,7 +19,7 @@ Location: {{ page.location }}
 
 ### Must Have
 - Excellent skills with HTML5& CSS3.
```
Comment (kevinwallimann) [inline _careers/ui_eng.md:21]: One last whitespace: `HTML5& CSS3` -> `HTML5 & CSS3`

Comment (Zejnilovic) [reply _careers/ui_eng.md:21]:
```suggestion
- Excellent skills with HTML5 & CSS3.
```

**Review by lokm01 — APPROVED** [review]
LGTM.

We should put in a "How to Apply" section, but lets figure that out separately.

**Review by kevinwallimann — APPROVED** [review]

**Pull request merged**